src/nni_manager/training_service/reusable/environments/openPaiEnvironmentService.ts

```diff
@@ -122,7 +122,7 @@ export class OpenPaiEnvironmentService extends EnvironmentService {
                                     break;
                                 case 'SUCCEEDED':
                                 case 'FAILED':
```

Comment:
add a break above Failed, so that succeeded env won't fail experiment.

Reply:
updated.